login-page.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=1920px, initial-scale=1.0">
+        <link rel="stylesheet" href="css/login-page.css">
+        <title>LOGIN</title>
+        <link rel="icon" href="images/favicon.ico">
+    </head>
+    <body class="login-container">
+        <div class="login-content">
+
+            <img src="images/login-page/aksu.png" alt="err">
```

Comment:
Always give 2 word names to files like images.
Only predefined files can have 1 name (ex "package.json", "README:md" or ".gitignore" ecc...)